runner/runners/queue.go

```diff
@@ -222,13 +222,20 @@ func (c *QueueController) Abort(run runner.RunID) (runner.RunStatus, error) {
 func (c *QueueController) abort(run runner.RunID) (runner.RunStatus, error) {
 	if run == c.runningID {
 		if c.runningAbort != nil {
-			log.WithFields(
-				log.Fields{
+			var fields log.Fields
+			if c.runningCmd != nil {
```

Comment:
Can we init log.Fields and then add to it? i.e. add c.runningID, then use a single 'if' to add runningCmd fields that drops the need for 'else'. (nit) Same for l287